src/client/components/menu/Menu.component.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+import './Menu.styles.css';
+import faUser from '../../assets/images/user-login.png';
+import faHeart from '../../assets/images/favorite-icon.png';
+import faShoppingCart from '../../assets/images/shopping-cart.png';
+
+export const Menu = (isAuthenticated) => {
```

Comment:
Right now `isAuthenticated` is assigned to the props object of the component, which looks either like this:

```
{
  isAuthenticated: false
}
```

Or this:

```
{
  isAuthenticated: true
}
```

By checking if `isAuthenticated` is true or false you are actually checking if the object is set or not - you're not evaluating against the property `isAutenticated`. To actually check the property inside the object you would need to check against `isAuthenticated.isAuthenticated` which is obviously very confusing.

You could solve it by either calling the props object `props`:

```suggestion
export const Menu = (props) => {
  ...
  props.isAuthenticated ? /* do this if true */ : /* do this if false */
  ...
}
```

Or alternatively (which I think is what you intended) you could use object destructuring to destructure the `isAuthenticated` property off the `props` object like so:

```suggestion
export const Menu = ({isAuthenticated}) => {
  ...
  isAuthenticated ? /* do this if true */ : /* do this if false */
  ...
}
```